client/src/components/JobCard/index.tsx

```diff
@@ -0,0 +1,85 @@
+import './style.css'
+import { useEffect, useState } from "react";
+import { Link } from 'react-router-dom';
+import { Grid, Box, Card, CardMedia, CardContent, Typography } from '@material-ui/core';
+import { Margin } from '@mui/icons-material';
+
+export const JobCard = (props: any) => {
+    // Constants
+    const BASE_CLEARBIT_URL = 'https://logo.clearbit.com/';
+
+    // State variables
+    const [url, setUrl] = useState(props.url);
+    const [company, setCompany] = useState(props.company);
+    const [role, setRole] = useState(props.role);
+    const [color, setColor] = useState(props.color);
```

Comment:
Are these redundant if the useEffect also sets them to the same value?

Reply:
The use effect is to update them if the value changes

Reply:
I also believe this is redundant